hack/gen-swagger-doc/Dockerfile

```diff
@@ -14,27 +14,32 @@
 
 FROM java:7-jre
 
-RUN apt-get update
-RUN apt-get install -qq -y asciidoctor
-RUN apt-get install -qq -y unzip
-RUN wget https://services.gradle.org/distributions/gradle-2.5-bin.zip
-RUN mkdir build/
-RUN unzip gradle-2.5-bin.zip -d build/
-
-RUN mkdir gradle-cache/
+RUN apt-get update && apt-get install -y \
+	asciidoctor \
+	unzip \
+	--no-install-recommends \
+	&& rm -rf /var/lib/apt/lists/*
```

Comment:
Nice cleanup of the downloaded stuff (rm -rf). Had not seen this before. Thanks! (http://askubuntu.com/questions/179955/var-lib-apt-lists-huge-in-12-04)